tests/other/metadce/mem_O3_STANDALONE_WASM.sent

```diff
@@ -0,0 +1,5 @@
+args_get
+args_sizes_get
+memory
+proc_exit
+table
```

Comment:
I thought `memory` and `table` were exported from the wasm module when building with WASM_STANDALONE?

Maybe that can be followup optimization.

Reply:
Good point, yeah, they are sent from the outside, but not received in the wasm. And metadce special-cases those so it doesn't clean them up. I'll optimize as a followup.